assets/section-image-banner.css

```diff
@@ -72,7 +70,46 @@
 }
 
 @media screen and (max-width: 749px) {
+  .banner--adapt:not(.banner--mobile-bottom) .banner__content {
+    height: 0;
+    overflow: hidden;
+  }
+
+  .banner--adapt:not(.banner--mobile-bottom) .banner__box {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+
+  .banner--mobile-bottom .banner__media,
+  .banner--stacked:not(.banner--mobile-bottom) .banner__media {
+    position: relative;
+  }
+
```

Comment:
Yeah, that's a side-effect of matching the container height to the first image height. Do you have any suggestions on how to fix that? 👀 

I honestly have no idea .-.

Reply:
I noticed this is also happening on desktop: https://screenshot.click/23-01-nsavn-03qb2.png
 I think we can just add the 39rem height to just media query of mobile: https://screenshot.click/23-02-0uxuj-2vltf.mp4

As for mobile, this is only an issue when you add the 39rem height when there are two images: https://screenshot.click/23-07-n1z2c-lxpio.mp4

I think you may want to remove it when there are two? 🤔

Reply:
Thank you so much for this detailed comment, Thomas. It helped a lot. 

I decided to rewrite my approach a little bit and add the `padding-top` to `content` on `mobile` (absolute positioning the `media`), and to `media` on `desktop` (absolute positioning the media).

It's MUCH better now, as the content will scale most of the times, except for:
- <img src="https://user-images.githubusercontent.com/37168033/130546419-db956e9a-80f3-40e2-9506-2d164f5604c2.png" width="250">
- <img src="https://user-images.githubusercontent.com/37168033/130546406-4ec87694-2550-4820-9008-7c1204141b1e.png" width="200">
- Long content inside `Heading Block`
- Wide images

But for that specific scenario, I still couldn't figure a solution, as the images CAN'T be absolute positioned (otherwise, it wouldn't follow `Adapt section height to first image size`) and therefore, the content needs to be `absolute` positioned to stay above the image.

That's the best I could do for now and I can't find a solution using CSS + Liquid. Not sure if this is considered an "edge case" or acceptable.

Reply:
To add context to the decision to not use an RTE here; we intentionally decided to allow merchants to add a lot of text to image with text and not to image banner. Since the content covers the image on the banner, we don't want to encourage merchants to add anything more than a few words. Because of this decision, if merchants wanted an opportunity to write more text and want it to be paired with an image they have the image with text section. The text block is disabled by default in the banner section for this reason as well.

Reply:
@sofiamatulis can you please share the combination of settings that led you to these screenshots? I couldn't reproduce that on my end. Here's what it looks like on mobile with:

![image](https://user-images.githubusercontent.com/37168033/130718137-d38bf051-fb64-4046-95c7-9fa72e9e3e89.png)


<img src="https://user-images.githubusercontent.com/37168033/130718082-53ed74f5-9dcc-4929-a22a-7f0b23b524dd.png" width="270">
<img src="https://user-images.githubusercontent.com/37168033/130718160-58476729-c5bc-4b11-81be-5de1a93a9c2d.png" width="270">